osama-breakout-room/qays/index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <script src="https://kit.fontawesome.com/9016f6ba0a.js"></script>
+</head>
+
+<body>
+
+    <header>
+        <div class="wrapper">
+            <div class="logo">
+                <img src="/osama-breakout-room/imgs/logo.png" alt="logo">
+            </div>
+            <nav>
+                <a class="btn btn-bordered " href="#">
+                    LOGIN
+                </a>
+                <a class="btn btn-blue" href="#">
+                    REGISTER NEW ACCOUNT
+                </a>
+            </nav>
+        </div>
+    </header>
+
+    <div class="wrapper">
+        <main>
+            <img class="profile-cover-container" src="/osama-breakout-room/imgs/cover.jpg" alt="cover">
+            <div class="profile">
+                <div class="profile-image-container">
+                    <img src="/osama-breakout-room/imgs/profile.jpg" alt="profile">
+                </div>
+                <div class="profile-info-container">
+                    <h2 id="name">John Al-Iraqi</h2>
+                    <h4 id="location"> <i class="fas fa-map-marker-alt"></i> North Iraq</h4>
```

Comment:
Try to always stick with classes rather than ids unless you really have to use an id.